lib/media/track/remotemediatrack.js

```diff
@@ -131,12 +151,47 @@ function mixinRemoteMediaTrack(AudioOrVideoTrack) {
         // to workaround: here disable the track when
         // there are no elements attached to it.
         this.mediaStreamTrack.enabled = false;
+
+        if (this._workaroundWebKitBug212780Cleanup) {
+          // unhook visibility change
+          this._workaroundWebKitBug212780Cleanup();
+          this._workaroundWebKitBug212780Cleanup = null;
+        }
       }
       return result;
     }
   };
 }
 
+function playIfPausedWhileInBackground(audioTrack) {
+  const { _log: log } = audioTrack;
+  let elementsPlaying = [];
+  function onVisibilityChanged() {
+    const isVisible = document.visibilityState === 'visible';
+    if (!isVisible) {
+      // when document goes to background, remember currently playing elements.
+      elementsPlaying = [...audioTrack._attachments.values()].filter(el => !el.paused);
+    } else {
+      // when document regains foreground restore playing elements.
```

Comment:
```suggestion
```

Reply:
I think this comment is useful

Reply:
Please prefix with `NOTE(mpatwardhan):`.